src/components/10-atoms/button/index.js

```diff
@@ -10,6 +10,10 @@ import buttonCSS from './index.scss';
 
 const ARROW_RIGHT = 'arrow-right';
 
+// equivalent to event.isTrusted. Unfortunatly IE11 does not support it
+const eventIsTrusted = e =>
+  !(!e.screenX && !e.screenY && !e.clientX && !e.clientY);
```

Comment:
Suspicious. Not obvious why this should be equivalent to isTrusted.

I suggest to try a new approach: What if you fire a custom event 'click' on the fakeButton and test for `event instanceOf CustomEvent`? Depends on this still successfully submitting in a form, of course. Worth a try, I think.

Reply:
done

Reply:
OK to stay in the approach implemented as-is, after our discussion